packages/crypto/package.json

```diff
@@ -5,10 +5,7 @@
   "type": "module",
   "scripts": {
     "lint": "eslint \"**/*.ts*\"",
-    "test:browser": "vitest run --config browser.config.ts",
-    "test:browser-watch": "vitest --config browser.config.ts",
-    "test": "vitest run",
-    "test-watch": "vitest"
+    "test": "vitest run"
```

Comment:
How do these tests know to run in a browser? Way back we had issues with the global `crypto` package being used in a nodejs environment.

Reply:
package-level config runs it in playwright

https://github.com/penumbra-zone/web/blob/ea1c3740e030a29705cdf3ac5e4dbd0c15e6cefe/packages/storage/vitest.config.ts#L1-L14